net.lutris.Lutris.yml

```diff
@@ -571,10 +571,10 @@ modules:
     sources: *sdl12_compat_sources
 
   # Needed for minecraft
-  - name: java11
+  - name: java17
```

Comment:
Did you check that, indeed, you can run minecraft with this version?

Reply:
But I try it
Normally you need 4 Java versions to run all Minecraft versions.
I think it was so:
|Minecraft Version|Java Version|
|-|- |
|<1.17|8|
|<1.20|17|
|>1.20|21|

Reply:
Newes version works great, so